ChangeLog

```diff
@@ -1,14 +1,15 @@
 libzim 7.0.0
 ============
 
-The version 7.0.0 is a major release.
+Version 7.0.0 is a major release.
 
-It rewrite the whole API. The changelog is not the place to
-describe the api change. A complete description of the API can be
-found in the documentation.
+The API has been completely rewritten.
+Most notable change is that namespaces are now hidden.
+The new API is described in documentation, which includes a Transition Guide from v6.
+
+ZIM files created with libzim 7 uses new ZIM major version (see Header section of spec).
```

Comment:
No. It now always use the major version 6, but it was already in use with major version 5 (depending if we use
extended cluster or not). We haven't move to another major version else the backward compatibility would have been lost.

We update the minor version from 0 to 1.

Reply:
Ah ok, let me fix that quick